source/model/Activator.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+
+
+class Activator(ABC):
+
+    def __init__(self, id):
+        self.activatorId = id
+
+    @property
+    @abstractmethod
+    def name(self):
+        """ The name of the activator which will be shown on the button"""
+        pass
+
+    @property
+    @abstractmethod
+    def stateType(self):
```

Comment:
Should the state not be an object in itself, which has a type? It seems like this class should not have to be concerned with it.